edunext_openedx_extensions/manage_api/views.py

```diff
@@ -143,7 +143,17 @@ def get(self, request, **kwargs):  # pylint: disable=unused-argument
 
     def _get_user(self, username, email):
         """
-        Helper method to return the username and email of an existing user.
```

Comment:
this refactor makes no sense. You split the method into 2 but you are not separating any of the logic.The idea of my inital comment was to extract the lines inside of the try block.